src/index.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Russo+One&display=swap');
```

Comment:
- [ ] I couldn't find a way to navigate the cars:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/d91c6e9c-6909-4816-b0fe-970b541c154a)

3 cars are shown, but there should be 4 in this case:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/362f004f-7379-40ea-b50b-a6a764e3ae62)

However, I couldn't find a way to navigate to the fourth car and beyond.

Kindly, fix this issue so all your cars can be navigated. 👍